src/guiapi/src/st7789v.c

```diff
@@ -63,9 +65,9 @@
 #define CLR565_BLUE    0x001f
 
 //color conversion
-#define _COLOR_565(clr) color_to_565(clr)
+#define _COLOR_TO_565(clr) color_to_565(clr)
```

Comment:
macros with an underscore at the beginning are reserved
e.g.: https://wiki.sei.cmu.edu/confluence/display/c/DCL37-C.+Do+not+declare+or+define+a+reserved+identifier
please don't use them when doing cleanup of old code or writing new one